### ISSUE TITLE
Conflicts in dependencies on macOS with flake8

### ISSUE THREAD

**Issue by cbirkenbeul** [ChrisRBe/PP-P2P-Parser#490]
Just tried to use your script but I got dependency issues on macOS 12

```
teqqy@teqqBook ~> python3 --version
Python 3.8.9
```

```
ERROR: Cannot install flake8==3.9.1 and pycodestyle==2.8.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested pycodestyle==2.8.0
    flake8 3.9.1 depends on pycodestyle<2.8.0 and >=2.7.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by ChrisRBe** [comment]
Hi @cbirkenbeul,

Thank you for reporting this issue. Could you tell me how you are installing the dependencies? Are you installing them via Pipfile or via requirements.txt?

**Comment by cschindlbeck** [comment]
Can you test https://github.com/cschindlbeck/PP-P2P-Parser/tree/fix-python-3.8.13-dependencies , if that works for you?

**Comment by ChrisRBe** [comment]
should be fixed.
